### PR TITLE
Nl webren

### PR DESCRIPTION
**Pull request by Nathan-Ladwig**
Committed all of the changes I was working on today in the Wb Renovation file, and added all the files I created to go with them in the second commit.

### CODE REVIEW

File: spec/webRenovation/functional/func_webRenovation_spec.rb [hunk 1]
```diff
@@ -9,24 +9,47 @@
     expect(home_page).to be_on_page
   end
 
-  scenario 'Find A-Z databases', :read_only, :smoke_test do
+#Scenario to find A-Z databases under the "research" tab, standalone
```
Comment (jeremyf) [inline spec/webRenovation/functional/func_webRenovation_spec.rb]: Remove comment

File: spec/webRenovation/pages/reserve_a_room_page.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+      def on_page?
+        super &&
+        on_valid_url?
+      end
+
+      def on_valid_url?
+        last_opened_window = page.driver.browser.window_handles.last
+        page.driver.browser.switch_to_window(last_opened_window)
+        current_url == ('http://nd.libcal.com/#s-lc-box-2749-container-tab1')
```
Comment (jeremyf) [inline spec/webRenovation/pages/reserve_a_room_page.rb]: Create a constant that describes/explains the URL.

File: spec/webRenovation/pages/technology_lending.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+module WebRenovation
+  module Pages
+    # /personal
```
Comment (jeremyf) [inline spec/webRenovation/pages/technology_lending.rb]: Is this comment accurate?

File: spec/webRenovation/pages/reserve_a_room_page.rb (hunk 2 above)
Comment (Nathan-Ladwig) [inline spec/webRenovation/pages/reserve_a_room_page.rb]: Created constant.

File: spec/webRenovation/pages/technology_lending.rb [hunk 4]
```diff
@@ -16,8 +14,12 @@ def on_valid_url?
         current_url == (Capybara.app_host + 'technology-lending')
       end
       def correct_content?
-        expect(page).to have_css('h2', text:'Technology and Miscellaneous Equipment Lending')
-        expect(page).to have_css('h3', text:'Contact Info', minimum: 1)
+        within('.container-fluid.content-area') do
+          find('h2', text:'Technology and Miscellaneous Equipment Lending')
+          find('h3', text: 'Contact Info', minimum: 1)
+        end
+        #expect(page).to have_css('h2', text:'Technology and Miscellaneous Equipment Lending')
```
Comment (h-parekh) [inline spec/webRenovation/pages/technology_lending.rb]: Remove commented lines if not needed.

Comment (h-parekh) [inline spec/webRenovation/pages/technology_lending.rb]: Looks like you fixed it in later commits. Make sure your subject line of a commit reflects the changes you are committing.

**Pull request merged**